examples/pages-router/next.config.js

```diff
@@ -22,6 +22,15 @@ const nextConfig = {
       ],
     },
   ],
+  redirects: [
+    {
+      source: "/redirect",
```

Comment:
This one also fails, page router use `trailingSlash: true`. It first get redirected to `/redirect/`, then it 404

Reply:
I have no idea why my change has this effect, I rather think this is another bug that now surfaces because of the e2e test 🤔. I see in `handleTrailingSlashRedirect` that this is not supported ATM? 

```
// Someone is trying to redirect to a different origin, let's not do that
    url.host !== "localhost" ||
```

Not sure how this should be tackled

Reply:
Nope not another bug, just noticed this `redirects` should be a function.
In the e2e test you also need to wait for the redirect and there is multiple text that match `Open source Next.js adapter` (in the right sidebar).
I'm pushing a fix